bouncer/views.py

```diff
@@ -39,6 +42,7 @@ def annotation(self):
         extension_url = "{uri}#annotations:{id}".format(
             uri=document_uri, id=annotation_id)
 
+        statsd.incr("views.annotation.200")
```

Comment:
I'd be inclined to call this `views.annotation` on its own. We'll get stats on statuses from gunicorn.